### ISSUE TITLE
Viewing a different camera while already connected to a camera skips the static

### ISSUE THREAD

**Issue by mirrorcult** [space-wizards/space-station-14#8680]
## Description

@vulppine is this intended behavior? seems wrong cause the idea behind static was to make the PVS pop-in not obvious

**Reproduction**

1. Refresh subnets
2. Click on a camera, see the static & 'Connecting' in the bottom left
3. Click on another camera once it's done, see that you skip the static load-in

**Screenshots**

https://user-images.githubusercontent.com/19853115/172076919-61d0686e-71d6-4e85-8abe-53ed522a1a8c.mp4

**Additional context**

**Comment by vulppine** [comment]
no, i'm not sure why this is occurring - a RemoveComp call is made during the update call in the client side system that deals with this, maybe it isn't actually being removed during that call?
